go/vt/vitessdriver/driver.go

```diff
@@ -209,8 +220,7 @@ func (c *conn) Rollback() error {
 }
 
 func (c *conn) Exec(query string, args []driver.Value) (driver.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), c.Timeout)
-	defer cancel()
+	ctx := context.TODO()
```

Comment:
I think semantically it may make more sense to use `context.Background()` here.

Reply:
That's what I initially started out with. But then, I realized that these functions are deprecated and must be eventually removed, which swayed me towards a TODO.

Reply:
:+1: